config-gammaray.h.cmake

```diff
@@ -3,9 +3,18 @@
 // relative install dirs
 #ifdef Q_OS_ANDROID
 # define GAMMARAY_PLUGIN_INSTALL_DIR "lib"
+#elif defined(GAMMARAY_INSTALL_QT_LAYOUT)
+# define GAMMARAY_PLUGIN_INSTALL_DIR "${PROBE_PLUGIN_INSTALL_DIR}"
```

Comment:
Since this is in the elseif branch of the Android branch above, this will never be hit on Android, is that intended?